### DISCUSSION TITLE
solvation free energy calculation via vertical energy gap

### DISCUSSION THREAD

**Discussion by howlong21** [deepmodeling/deepmd-kit#2531] [Q&A]
Hi all,
I am trying to calculate the solvation free energy of cations with deep learning potential, using the vertical energy gap method as described in https://doi.org/10.1063/5.0098330. The vertical energy gap method has to set the scale factor for systems with and without the solute, respectively. 
I have trained a LiCl solution model (i.e., graph.pb), and it's convenient to combine it with lj/cut potential in LAMMPS by pair_style hybrid/scaled. For example,
pair_style hybrid/scaled 1.0 deepmd graph.pb 0.0 lj/cut 2.5
pair_coeff      * * deepmd
pair_coeff     * * lj/cut 1.0 1.0

But i have no idea to combine two deep learning potential models (one model for LiCl solution and the other for solution without Li) together with different scale factors, which is needed in the calculation of solvation free energy.  Does anyone know how to implement this in LAMMPS?

Thanks in advance

**Comment by wanghan-iapcm** [comment]
You may try fix adapt/fep, which may allow you assigning different prefactors on different models.
